### PR TITLE
aci_aaa_login_domain

### PR DESCRIPTION
**Pull request by schunduri**

### CODE REVIEW

**Comment by schunduri** [comment]
Need to add Login domain url to aci.py:

url: '/api/mo/uni/userext/logindomain-' + login_domain + '.json'

File: library/aci_aaa_login_domain.py [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+  I(aaa:LoginDomain) at U(https://developer.cisco.com/media/mim-ref/MO-aaaLoginDomain.html).
+author:
+- Swetha Chunduri (@schunduri)
+- Dag Wieers (@dagwieers)
+- Jacob McGill (@jmcgill298)
+version_added: '2.4'
+requirements:
+    - ACI Fabric 1.0(3f)+
+options:
+    login_domain:
```
Comment (dagwieers) [inline library/aci_aaa_login_domain.py]: This should be parameter **name** with alias **login_domain**.

File: library/aci_aaa_login_domain.py [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+       description:
+       - Domain name
```
Comment (dagwieers) [inline library/aci_aaa_login_domain.py]: The description is not an English sentence explaining what the parameter is for.
Should end with a dot.

File: library/aci_aaa_login_domain.py [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+       required: true
```
Comment (dagwieers) [inline library/aci_aaa_login_domain.py]: Make this: `required: yes`

File: library/aci_aaa_login_domain.py [hunk 4]
```diff
@@ -0,0 +1,141 @@
...
+    description:
+       description:
+       - Description for Login Domain
```
Comment (dagwieers) [inline library/aci_aaa_login_domain.py]: Should become: `The description for the Login Domain.`
Should end with a dot.

File: library/aci_aaa_login_domain.py [hunk 5]
```diff
@@ -0,0 +1,141 @@
...
+extends_documentation_fragment: aci
+'''
+
+EXAMPLES = r'''
+- name: Add a new login domain
+  aci_aaa_login_domain:
+    hostname: apic
+    username: admin
+    password: SomeSecretPassword
+    login_domain: fallback
```
Comment (dagwieers) [inline library/aci_aaa_login_domain.py]: Is "fallback" an actual Login Domain ? In you description you stated this has to be a "Domain Name" ?

File: library/aci_aaa_login_domain.py (hunk 1 above)
Comment (jmcgill298) [reply library/aci_aaa_login_domain.py]: we have been aliasing name in the other modules

Comment (dagwieers) [reply library/aci_aaa_login_domain.py]: Sorry, you're correct. The alias **name** was missing. Thanks !

**Comment by jmcgill298** [comment]
@schunduri to have the URLs built, you just need to update the mapping dictionary in module_utils/aci.py
```
URL_MAPPING = dict(
    ...
    login_domain=dict(aci_class='aaaLoginDomain', mo='userext/logindomain-', key='name'),
    ...
```

**Comment by dagwieers** [comment]
Please move everything to upstream.

**Pull request closed without merging**